src/secondPackageForHomework6/CheckCalculator.java

```diff
@@ -0,0 +1,13 @@
+package secondPackageForHomework6;
```

Comment:
package name should be like secondpackageforhomework6

Reply:
Changed the package name to secondpackageforhomework6